kalitectl.py

```diff
@@ -571,6 +572,27 @@ def start(debug=False, watch=False, daemonize=True, args=[], skip_job_scheduler=
             'server.socket_port': port
         })
 
+    # idealy we'd have a setting element, but these are imported much later
+    # and it will require a much deeper change
+    idle_timeout = int(os.getenv("KALITE_IDLE_TIMEOUT", 0))
```

Comment:
to be complete, 0 is falsy and int() catches all 'invalid' values to 0, that is falsy. the change you're suggesting leads to a much more complicated handeling.